src/lib/svg2component.js

```diff
@@ -12,7 +12,12 @@ module.exports = (name, svg, isTypeScriptOutput) => {
   toReactAttributes($svg, $);
   const children = $svg.html();
   const viewBox = $svg.attr('viewBox');
-  const iconJsx = `<Icon viewBox="${viewBox}" {...props}>${children}</Icon>`;
+  const viewBoxAttr = viewBox ? `viewBox="${viewBox}" ` : '';
```

Comment:
code duplication ahead! :)
can you just make it a little bit nicer? entries/map/reduce/etc...

Reply:
Looks fine to me, it would be more confusing with reduce.

Reply:
Not too confusing I guess.
```
const attrs = ['viewBox', 'width', 'height'].map(name => {
  const value = $svg.attr(name);
  return value ? `${name}="${value}"` : '';
}).join(' ');
```